Makefile

```
@@ -511,6 +512,14 @@ endif
 endif
 endif
 
+ifeq ($(ENABLE_GHDL),1)
+GHDL_DIR ?= /usr/local
+GHDL_INCLUDE_DIR ?= $(GHDL_DIR)/include
+GHDL_LIB_DIR ?= $(GHDL_DIR)/lib
+CXXFLAGS += -I$(GHDL_INCLUDE_DIR) -DYOSYS_ENABLE_GHDL
+LDLIBS += $(GHDL_LIB_DIR)/libghdl.a $(shell cat $(GHDL_LIB_DIR)/libghdl.link)
```

This would be more portable:
```suggestion
LDLIBS += $(GHDL_LIB_DIR)/libghdl.a $(file <$(GHDL_LIB_DIR)/libghdl.link)
```
(Yes, I know MinGW is needed anyway, but the existing Makefile is so horrifically slow due to forking lots of crap that you need something like kati to get a reasonable warm build time, so no need to make stuff even worse.)

---

I was not aware of this possible improvement. Thanks!